cpp/src/Ice/OutputStream.cpp

```diff
@@ -162,11 +161,11 @@ void
 Ice::OutputStream::initialize(Instance* instance, const EncodingVersion& encoding)
 {
     assert(instance);
-
-    _instance = instance;
     _encoding = encoding;
 
-    _format = _instance->defaultsAndOverrides()->defaultFormat;
+    _format = instance->defaultsAndOverrides()->defaultFormat;
```

Comment:
Shouldn't this be a second constructor? it is not clear why we use a `initialize` method.

Reply:
I will clean this up in a follow-up PR.

`initialize` is used by the IceDB template code, but I think we can remove it.